src/Projection/ProjectionManager.php

```diff
@@ -22,13 +22,13 @@ public function createQuery(): Query;
     public function createProjection(
         string $name,
         array $options = []
-    ): Projector;
+    ): MetadataAwareProjector;
```

Comment:
Oops, still a bc break... 😅

Reply:
I had too much going on today... Give me an hour or two...

Reply:
Although, I may say I'm testing the reviewer :D

Reply:
You're right. I had a quick look, but it was too quick.

Reply:
anyone worked with https://github.com/Roave/BackwardCompatibilityCheck?

Reply:
@basz I can give it a look and provide a PR to check it out ;)